sensor_msgs/PointCloud2 full_reconstructed_pcd
---
geometry_msgs/PoseArray local_basis